lib/models/discount.freezed.dart

```diff
@@ -1,5 +1,6 @@
 // coverage:ignore-file
 // GENERATED CODE - DO NOT MODIFY BY HAND
+// ignore_for_file: type=lint
```

Comment:
These `.freezed.dart` were not updated to the latest version. I am going to change CI to make sure these are updated everytime something gets merged to `main`.

Reply:
https://github.com/RevenueCat/purchases-flutter/pull/302 😬

Reply:
nice!